ui/eureka/eureka.js

```diff
@@ -3621,7 +3622,9 @@ class EurekaTracker {
     // Adds field note drops, name, id & rarity of those
     if (this.zoneInfo.treatNMsAsSkirmishes && this.options.EnrichedSTQ && nm.fieldNotes) {
       for (const note of fieldNotesList) {
-        if (note.id === nm.fieldNotes)
+        if (typeof nm.fieldNotes === 'string')
+          nmNote = [note];
```

Comment:
Where is `nmNote` declared?

Reply:
oh wow
I need to some doing PRs while half-asleep